app/views/layouts/course_settings.html.slim

```diff
@@ -0,0 +1,10 @@
+= render within_layout: controller.parent_layout(of_layout: 'course_settings') do
+  ul.nav.nav-tabs role="tablist"
+    - controller.settings.each do |item|
+      - if item[:controller].to_s == params[:controller]
```

Comment:
there's a way to check this... without using `params[:controller]` let me go find.